src/main/resources/default/templates/biz/jobs/job.html.pasta

```diff
@@ -131,7 +131,7 @@
                         sirius.dispatchEvent("clear", _element);
                     }
                     if (typeof param.updatedValue !== 'undefined') {
-                        sirius.dispatchEvent("updated-value", _element, {detail: param.updatedValue});
+                        sirius.dispatchEvent("updated-value", _element, param.updatedValue);
```

Comment:
Das ist Absicht?

Reply:
Yes, daher auch in nem extra commit

Reply:
Ich hab das event dispatching relativ spät noch von jquery auf sirius core umgestellt, und bei der Migration verpasst, dass es in sirius core schon passend gewrapped wird. Das ist mir leider erst kürzlich aufgefallen